<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JIDSearch</class>
 <widget class="QWidget" name="JIDSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="grpSearch">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="JIDJabberSearch" name="jidSearch" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnAdvanced">
     <property name="text">
      <string>&amp;Advanced</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="Spacer2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btnBrowser">
     <property name="text">
      <string>Jabber &amp;browser</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JIDJabberSearch</class>
   <extends>QWidget</extends>
   <header>jabbersearch.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnAdvanced</tabstop>
  <tabstop>btnBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
